### ISSUE TITLE
Require whitespace inside inline array literals

### ISSUE THREAD

**Issue by thasner** [palantir/tslint#1045]
good

``` ts
const x = [ "one", "two" ];
```

bad

``` ts
const x = ["one", "two"];
```

**Comment by thasner** [comment]
similar to #1044

**Comment by buzinas** [comment]
This rule was added to [TSLint ESLint Rules](/buzinas/tslint-eslint-rules) v1.2.0

**Comment by adidahiya** [comment]
Closing this with the same reason as https://github.com/palantir/tslint/issues/1044#issuecomment-466843067
